pkg/operator/ceph/csi/spec.go

```diff
@@ -137,13 +138,13 @@ var (
 // manually challenging.
 var (
 	// image names
-	DefaultCSIPluginImage   = "quay.io/cephcsi/cephcsi:v3.8.0"
+	DefaultCSIPluginImage   = "quay.io/cephcsi/cephcsi:v3.9.0"
 	DefaultRegistrarImage   = "registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.7.0"
```

Comment:
can you also check any sidecar images need to be updated?

Reply:
> can you also check any sidecar images need to be updated?

I'll update it in a follow-up pr, 
it'll make it easier to review this pr.

Reply:
opened tracker https://github.com/rook/rook/issues/12442